### DISCUSSION TITLE
Too many connections error when iteratiing throuhg array with update() method

### DISCUSSION THREAD

**Discussion by czuforb** [prisma/prisma#5487] [Q&A]
Hi!

I would like to update multiple rows from the database periodically. I want to make this by fetching every row I need, then iterate through each of them and update a specific value. I have to fetch the rows beforehand because I need a specifig unique value to find the updateable row.

What is the best way to do this?

I managed to solve this by, chaining a findMany and an update function. **The problem is that after a few iterations, I'll get an error that there are too many connections.** I tried to use $disconnect function, but doesn't seems to work.

Thank you.

**Comment by janpio** [comment]
Can you share a reproduction? We would need to look into the details what is going on inside of Prisma to understand this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
